port/unix/omrsl.c

```diff
@@ -438,3 +439,79 @@ omrsl_startup(struct OMRPortLibrary *portLibrary)
 {
 	return 0;
 }
+
+/**
+ * Native Libraries
+ *
+ * This function is called go get all the shared libraries loaded by a process.
+ *
+ * @param[in] portLibrary Pointer to the OMR port library.
+ * @param[in] callback Function to be called for each library.
+ * @param[in] userData User-defined data passed to the callback.
+ *
+ * @return 0 if successful, or the first non-zero return value from the callback.
+ */
+uintptr_t
+omrsl_get_libraries(struct OMRPortLibrary *portLibrary, OMRLibraryInfoCallback callback, void *userData)
+{
+#if defined(LINUX)
+/* Length of buffer. */
+#define READ_CHUNK_SIZE (PATH_MAX + 100)
+	char buffer[READ_CHUNK_SIZE + 1];
```

Comment:
Please remove `READ_CHUNK_SIZE` (it's only used once).
Please add a comment explaining why we're adding 101 to `PATH_MAX`.